packages/tizen_bundle/lib/tizen_bundle.dart

```diff
@@ -0,0 +1,320 @@
+// Copyright 2022 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ffi';
+
+import 'package:ffi/ffi.dart';
+import 'package:tizen_interop/6.5/tizen.dart';
```

Comment:
Is the API only supported by Tizen 6.5 or above? As far as I can see [TizenFX's Bundle class](https://samsung.github.io/TizenFX/API4/api/Tizen.Applications.Bundle.html) was added in Tizen 4.0 and no API has been added since then.

Reply:
It's not supported by Tizen 6.5. The bundle API is  supported since Tizen 2.x.
We can change it to "4.0".

Reply:
Yes, please update this line and `tizen-manifest.xml`. Plugins should always target the lowest possible Tizen version as they can.

Reply:
Thanks. I've updated it.